src/StatusQ/Controls/StatusAccountSelector.qml

```diff
@@ -126,153 +138,150 @@ Item {
         font.pixelSize: 13
         height: txtAssetBalance.height
     }
-    StatusSelect {
-        id: select
+
+    StatusComboBox {
+        id: comboBox
+
         label: root.label
-        model: root.accounts
         width: parent.width
-        menuAlignment: StatusSelect.MenuAlignment.Left
-        selectMenu.delegate: menuItem
-        selectMenu.width: dropdownWidth
-        selectedItemComponent: Item {
-            anchors.fill: parent
+
+        model: SortFilterProxyModel {
+            sourceModel: root.accounts
+            filters: [
+                ValueFilter {
+                    roleName: "walletType"
```

Comment:
hey doesnt this make you feel like this doesnt belong on StatusQ side, it depends on BE values, when those change this will have to be updated too. However doesnt have to be fixed in this PR.